README.md

```diff
@@ -45,6 +45,16 @@ Options:
   -a, --append                Turns on append results to file mode rather than replace
 ```
 
+### Examples:
+  Test the latest underscore module:
+    `citgm underscore@latest`
+
+  Test a local module:
```

Comment:
@MylesBorins turns out it is working, it was just a strange error message that was confusing me before (basically if you do a `citgm ./deps/npm` it fails unless you first `npm i -g rimraf`).